content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,79 @@ description: Istio 1.5 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
+
+- **Improved** performance of `ServiceEntry` updates #19305.
+- **Improved** Envoy sidecar readiness probe to more accurate determine readiness #18164.
+- **Improved** performance of configuration updates by sending partial updates where possible #18354.
+- **Added** an option to control locality load balancer settings for each Service #18406.
+- **Fixed** an issue where pods crashing would trigger excessive configuration pushes #18574.
+- **Fixed** issues with applications that call themselves #19308.
+- **Added** detection of `iptables` failure when using Istio CNI #19534.
+- **Added** `consecutive_5xx` and `gateway_errors` as outlier detection options #19771.
+- **Improved** `EnvoyFilter` matching performance #19786.
+- **Added** support for `HTTP_PROXY` protocol #19919.
+- **Improved** `iptables` setup to use `iptables-restore` by default #18847.
+- **Improved** Gateway performance by filtering unused clusters. This setting is disabled by default #20124.
```

Comment:
```suggestion
- **Dark launched** Gateway performance by filtering unused clusters via PILOT_FILTER_GATEWAY_CLUSTER_CONFIG., which is disabled by default #20124.
```

Reply:
I think people outside the project would be confused by this. What is "dark launched" and how do I configure "via PILOT_FILTER_GATEWAY_CLUSTER_CONFIG"?

Reply:
ideally a link to how to further explain this would be good, but i don't find it in our doc.